dts/arm/atmel/sam3x.dtsi

```diff
@@ -25,7 +25,7 @@
 
 	soc {
 		uart0: uart@400E0800 {
-			compatible = "atmel,sam3x-uart";
+			compatible = "atmel,sam-uart";
 			reg = <0x400E0800 0x124>;
 			interrupts = <8 0>;
 			status = "disabled";
```

Comment:
On the SAM3X there is only one UART and 4 USARTs. I will add the 4 USARTs to the file sam3x.dtsi.